build/server-error.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-env node */
+const React = require('react');
+const RedBox = require('redbox-react').RedBoxError;
+const ReactDOMServer = require('react-dom/server');
+
+function renderError(res, error) {
+  res.write(
+    '<!DOCTYPE html><html><head><title>Server render error</title></head><body>'
```

Comment:
Technically I think this is not a server render error, but a server startup error